Extensions/dnSpy.Debugger/dnSpy.Debugger/ToolWindows/CodeBreakpoints/CodeBreakpointVMComparer.cs

```diff
@@ -37,7 +37,8 @@ sealed class CodeBreakpointVMComparer : FormatterObjectVMComparer<CodeBreakpoint
 				return x.Order - y.Order;
 
 			if (Tag == PredefinedTextClassifierTags.CodeBreakpointsWindowName)
-				return Comparer<DbgCodeLocation>.Default.Compare(x.CodeBreakpoint.Location, y.CodeBreakpoint.Location);
+				//TODO
```

Comment:
Need your help here. I cannot get to the "visible" name of the condition

Reply:
I assume you still need help despite the fact that github says that your comment is outdated.

Have a look at the formatter classes to see how the text is formatted. Each sub dir has one.